pagopa/io-app
# feat: [#174034081] Unlock code is persisted across logins if citizen doesn't change

## Short description
It is now possible to log-out and log back in without resetting the unlock code, unless the SPID user is different.

## How to test
- Log-in and set an unlock code
- Log-out
- Log back in, no unlock code reset should be asked
- Log-out
- In `mock-server/src/global.ts` change the fiscal code and restart the mock server
- Log back in, you should be prompted to set a new unlock code
